StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetAppearance.swift

```diff
@@ -329,6 +329,9 @@ public extension PaymentSheet.Appearance {
             /// - Note: If `nil`, uses a default font based on `appearance.font`
             public var titleFont: UIFont?
 
+            /// The font of the subtitle in a payment method row e.g. "Buy now or pay later with Klarna"
+            public var subtitleFont: UIFont?
```

Comment:
Should we add a note similar to what we do for `titleFont` when it's nil?
`/// - Note: If `nil`, uses a default font based on `appearance.font``